roles/postgresql/tasks/main.yml

```diff
@@ -19,6 +19,16 @@
   shell: "createdb {{ lookup('env','USER') }}"
   when: postgresql_user_database_exists.rc == 1
 
+- name: Check if postgres user exists
+  shell: "psql -t -c '\\du' | cut -d \\| -f 1 | grep -qw postgres"
```

Comment:
Is it possible to use grep -c option to avoid ignored error?

Reply:
I'm not 100% sure but I think grep is running silently because ansible uses the return code and not the standard output content